rio_stac/scripts/cli.py

```diff
@@ -0,0 +1,100 @@
+"""rio_stac.scripts.cli."""
+import datetime
+import json
+
+import click
+from pystac.utils import datetime_to_str, str_to_datetime
+from rasterio.rio import options
+
+from rio_stac import create_stac_item
+
+
+def _cb_key_val(ctx, param, value):
+    if not value:
+        return {}
+    else:
+        out = {}
+        for pair in value:
+            if "=" not in pair:
+                raise click.BadParameter(
+                    "Invalid syntax for KEY=VAL arg: {}".format(pair)
+                )
+            else:
+                k, v = pair.split("=", 1)
+                out[k] = v
+        return out
+
+
+@click.command()
+@options.file_in_arg
+@click.option(
+    "--datetime",
+    "-d",
+    "input_datetime",
+    type=str,
+    help="The searchable date and time of the assets, in UTC.",
+)
+@click.option(
+    "--extension",
+    "-e",
+    type=str,
+    default=["proj"],
```

Comment:
Is there a way to exclude the proj extension with this flag?

Reply:
no! 🤷‍♂️

Reply:
@geospatial-jeff if you pass any other extension, the defaults will be overwritten. 

maybe we can accept `--extension ""` to remove any extensions!

Reply:
done in https://github.com/developmentseed/rio-stac/pull/3/commits/3470127048873b9400b1a02ec3db87358c40dbe7